### PR TITLE
refactoring(Users): ready for test

### PR DESCRIPTION
**Pull request by DiLiUA**

### CODE REVIEW

File: src/auth0/auth.service.ts [hunk 1]
```diff
@@ -43,11 +42,11 @@ export class AuthService implements OnDestroy {
               private userService: UserService) {
     this.isAuthenticated();
 
-    this.userService.updateUserAccount.subscribe((res: boolean): void | undefined => {
+    this.userService.updateUserAccount.subscribe((res: boolean): void => {
```
Comment (Ugmaxie) [inline src/auth0/auth.service.ts]: you can use filter here to avoid next "if"

File: src/event-launch/event-launch.component.ts [hunk 2]
```diff
@@ -13,13 +13,12 @@ import { customToastOptions } from '../shared/models/toasty-options.model';
 import { FileConfig } from '../shared/file-uploader/fileUploader.interface';
 import { FileUploaderComponent } from '../shared/file-uploader/file-uploader.component';
 import { FieldConfig } from '../shared/multiple-inputs/fieldConfig.interface';
-import { User } from '../user-service/user.model';
+import { User, UserService } from '../shared/services/user-service';
 import { EventInfo } from '../shared/event-info/event-info.interface';
 import { NewStreamModel } from '../shared/wowza-streaming-cloud/new-stream.model';
 import { Pattern, CrowdcampaignForSomeOtherArtist, CrowdcampaignForMySelf, CrowdCampaignType } from '../enums';
-import { OembedService } from '../shared/servises/oembed/oembed.service';
+import { OembedService } from '../shared/services';
```
Comment (Ugmaxie) [inline src/event-launch/event-launch.component.ts:20]: Can 16 and 20 strings be imported together?

File: src/shared/services/user-service/user.model.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
-import { environment } from '../environments/environment';
-import { RouterLinks } from '../enums';
-import { City, Country } from '../shared/models';
+import { environment } from '../../../environments/environment';
+import { RouterLinks } from '../../../enums/index';
```
Comment (Ugmaxie) [inline src/shared/services/user-service/user.model.ts]: you shouldn't use "index" in this case

File: src/shared/services/user-service/user.model.ts [hunk 4]
```diff
@@ -1,6 +1,6 @@
...
+import { City, Country } from '../../models/index';
```
Comment (Ugmaxie) [inline src/shared/services/user-service/user.model.ts]: the same

File: src/shared/services/user-service/user.service.ts [hunk 5]
```diff
@@ -3,11 +3,12 @@ import { HttpClient } from '@angular/common/http';
...
 
-import { Config } from '../app.config';
-import { ParsedProfile } from '../auth0/parsed-profile.interface';
-import { User } from './user.model';
-import { UpdateUserProfileRequestObject, UpdateUserProfileResponseObject } from './update-user-profile.interface';
-import { Notifications } from '../my-profile/notification.model';
+import { Config } from '../../../app.config';
+import { ParsedProfile } from '../../../auth0/parsed-profile.interface';
+import { Notifications } from '../../../my-profile/notification.model';
+import { User, UpdateUserProfileRequestObject, UpdateUserProfileResponseObject } from './index';
```
Comment (Ugmaxie) [inline src/shared/services/user-service/user.service.ts]: you shouldn't use index file

**Review by Ugmaxie — CHANGES_REQUESTED** [review]

File: src/auth0/auth.service.ts (hunk 1 above)
Comment (DiLiUA) [reply src/auth0/auth.service.ts]: it makes no sense

File: src/event-launch/event-launch.component.ts (hunk 2 above)
Comment (DiLiUA) [reply src/event-launch/event-launch.component.ts:20]: no. that's because of `user-service` has own `index.ts file`

**Review by Ugmaxie — APPROVED** [review]

**Pull request merged**